src/validator.cpp

```diff
@@ -368,11 +368,11 @@ void Validator::swap(Validator &rhs)
 }
 
 bool pathIsRelative(const std::string &path) {
-    // KRM TODO Not sure if this should be in this file or in a utilities file?
-
+    // TODO Not sure if this should be in this file or in a utilities file?
     // Starting with . or .. in any operating system implies relative path
-    if (path.at(0) == '.') 
-        return true;
+    if(path.size())
+        if (path.at(0) == '.') 
+            return true;
 #ifdef _WIN32
```

Comment:
If it is solely a 64 bit consideration then it should be _WIN64 otherwise _WIN32.

Have I ever mentioned how much I hate #ifdef to exclude code?  I'm going to have to go through the guidelines on this one.